### ISSUE TITLE
which version of Atheros X NIC work properly?

### ISSUE THREAD

**Issue by xiaolingu** [xieyaxiongfly/Atheros-CSI-Extraction-TOOL-UserSpace-APP#5]
Hi 
According to your knowledge about users' response to your CSI-tool in email and so on, Which version of Atheros NIC can work successfully in practice ? 
Looking forward to your reply !

Nanjing
@xieyaxiongfly

**Comment by xieyaxiongfly** [comment]
Hi,

Till now, the following version has been proven working successful. 

AR9590
AR9580
AR9565
AR9462
AR9380

Thanks

**Comment by xieyaxiongfly** [comment]
And you can find the information at there
http://pdcc.ntu.edu.sg/wands/Atheros/DeviceList.html

**Comment by xiaolingu** [comment]
Thank you for your rapid response！！
In addition，I have another question whether I need to identify the BIOS in my laptop？My laptop likely
set a access for hardware equipment.How do you deal with the problem ?

**Comment by xieyaxiongfly** [comment]
Yes, you need to modify the BIOS in the laptop to bypass the hardware blacklist.

Sorry for answering late, coz I was traveling these days.

**Comment by xiaolingu** [comment]
Excuse me ， I have installed the CSI tool in a PC . However when using the read_csi file to read the specific file , CSI item doesn't include any data and CSI length == 0 .Should I use another computer including Atheros NIC as a Soft AP ? 

Thanks

**Comment by xieyaxiongfly** [comment]
Yes. You must have another PC installed with our tool and configured as AP.
